<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiCalculator</class>
 <widget class="QMainWindow" name="GuiCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuiCalculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QBackground{
	background-color: #696969;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QPushButton" name="Num7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="Num6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="subButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #008080;
	padding:10px
}

QPushButton:pressed{
	background-color: #00b3b3;
	border: 2px solid #008080;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="ChangeSign">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  	#003333;
	padding:8px;
	color: #ffffff;
}

QPushButton:pressed{
	background-color: #006666;
	border: 2px solid #003333;
	color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="Num4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="addButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #008080;
	padding:10px
}

QPushButton:pressed{
	background-color: #00b3b3;
	border: 2px solid #008080;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="Num0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="helpButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  	#696969;
	padding:8px;
	color: #ffffff;
}

QPushButton:pressed{
	background-color: #8c8c8c;
	border: 2px solid #696969;
	color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="Num2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="Num1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="sqrtButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #008080;
	padding:10px
}

QPushButton:pressed{
	background-color: #00b3b3;
	border: 2px solid #008080;
}</string>
      </property>
      <property name="text">
       <string>root(x)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="powerButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #008080;
	padding:10px
}

QPushButton:pressed{
	background-color: #00b3b3;
	border: 2px solid #008080;
}</string>
      </property>
      <property name="text">
       <string>x^y</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="3">
     <widget class="QPushButton" name="eqButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #008080;
	padding:10px
}

QPushButton:pressed{
	background-color: #00b3b3;
	border: 2px solid #008080;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="Num3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="mulButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #008080;
	padding:10px
}

QPushButton:pressed{
	background-color: #00b3b3;
	border: 2px solid #008080;
}</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="divButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #008080;
	padding:10px
}

QPushButton:pressed{
	background-color: #00b3b3;
	border: 2px solid #008080;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="floatButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Num9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Num8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="logButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #008080;
	padding:10px
}

QPushButton:pressed{
	background-color: #00b3b3;
	border: 2px solid #008080;
}</string>
      </property>
      <property name="text">
       <string>ln(x)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="facButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #008080;
	padding:10px
}

QPushButton:pressed{
	background-color: #00b3b3;
	border: 2px solid #008080;
}</string>
      </property>
      <property name="text">
       <string>x!</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Num5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #D3D3D3;
	border-radius:10px;
	color: black;
	font-size: 120%;
}

QPushButton:pressed{
	background-color: #808080;
	border: 2px solid #D3D3D3;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="clearButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  	#003333;
	padding:8px;
	color: #ffffff;
}

QPushButton:pressed{
	background-color: #006666;
	border: 2px solid #003333;
	color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLightmode">
   <property name="text">
    <string>Lightmode</string>
   </property>
  </action>
  <action name="actionDarkmode">
   <property name="text">
    <string>Darkmode</string>
   </property>
  </action>
  <action name="actionHelp_me">
   <property name="text">
    <string>Help me</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
